packages/fuchsia_remote_debug_protocol/lib/src/fuchsia_remote_connection.dart

```diff
@@ -506,6 +507,30 @@ class FuchsiaRemoteConnection {
     _pollDartVms = true;
   }
 
+  /// Helper for getDeviceServicePorts() to extract the vm_service_port from
+  /// json response.
+  List<int> getVmServicePortFromInspectSnapshot(dynamic inspectSnapshot) {
```

Comment:
Per style guide, avoid using `dynamic` https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-using-var-and-dynamic

At worst, use `Object?`, but it seems like we are requiring this to be as a list of maps, so let's just make the parameter type specific to avoid later casts, and then add casts if necessary to the places where we invoke this function.